docs/modules/sql/pages/user-defined-types.adoc

```diff
@@ -625,4 +547,7 @@ Updating nested UDT column with Query Parameter:
 [source,java]
 ----
 hz.getSql().execute("UPDATE users SET organization = ((organization).id, (organization).name, ?)", office);
-----
\ No newline at end of file
+----
+
+== Rolling Upgrade Notes
+You must drop all user-defined types and mappings with UDTs before the rolling upgrade, and recreate them with the new semantics after upgrading.
```

Comment:
add information about versions to which this restricion applies. This documentation will probably not be updated in 5.5 and later

Reply:
099f756d